tests/test_apt.py

```diff
@@ -0,0 +1,57 @@
+'''Define unit tests for parsing APT templates with pytest.
+
+Authors
+-------
+    - Lauren Chambers
+
+Use
+---
+    Ensure you have pytest installed. Then, simply run pytest in any
+    parent directory of nircam_simulator/tests/:
+    >>> pytest
+'''
+
+import os
+
+from nircam_simulator.scripts import write_observationlist, yaml_generator, utils
+from nircam_simulator.scripts.get_catalog import get_all_catalogs
+
+INSTRUMENTS = ['NIRCam']
+PROPOSAL_ID = '1111'
+
+TESTS_DIR = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def test_APT():
+    '''Parse the given APT files and create a set of .yamls for each
+    instrument in the INSTRUMENTS list.
+    '''
+    for instrument in INSTRUMENTS:
+        # Define .pointing and .xml file locations
+        pointing_file = os.path.join(TESTS_DIR, 'test_data', instrument + 'Test.pointing')
+        xml_file = os.path.join(TESTS_DIR, 'test_data', instrument + 'Test.xml')
+
+        # Point to appropriate output directory
+        out_dir = os.path.join(TESTS_DIR, 'test_data', 'APT_{}_out'.format(instrument))
+
+        # Create/locate catalogs for target(s)
+        sw_cats, lw_cats = get_all_catalogs(pointing_file, PROPOSAL_ID)
```

Comment:
Are the catalogs already present? If not this will try to access vizier which may fail if the machine running the test cannot access vizier. 
Tests should be as basic as possible, so here I would suggest to create the catalogs in advance and include them in the test_data directory.
You could write an independent test for the automatic catalog generation, but I am not sure how/if this will work once we run this on Travis or similar.

Reply:
Ah, I hadn't thought about machines that can't get to Vizier. I'll modify this to just pull the catalogs from the test_data directory, and I'll make an issue for writing a test for catalog creation.